src/System.Windows.Forms.Design/tests/UnitTests/ControlDesignerTests.cs

```diff
@@ -1,278 +1,460 @@
-        Assert.True(controlDesigner.ForceVisible);
-        Assert.Throws<InvalidOperationException>(controlDesigner.GetParentComponentProperty);
-        Assert.False(controlDesigner.SerializePerformLayout);
+
+        controlDesigner.AutoResizeHandles.Should().BeFalse();
+        controlDesigner.ControlSupportsSnaplines.Should().BeTrue();
+        controlDesigner.ForceVisible.Should().BeTrue();
+        controlDesigner.SerializePerformLayout.Should().BeFalse();
+
+        Action action1 = () =>
+        {
+            using Control _ = controlDesigner.Control;
+        };
+
+        Action action2 = () =>
+        {
+            using IComponent component = controlDesigner.Component;
+        };
+
+        Action action3 = () => controlDesigner.GetParentComponentProperty();
+
+        action1.Should().Throw<InvalidOperationException>();
+        action2.Should().Throw<InvalidOperationException>();
+        action3.Should().Throw<InvalidOperationException>();
     }
 
     [WinFormsFact]
     public void ControlDesigner_PropertiesTest()
     {
         using TestControlDesigner controlDesigner = new();
         using Button button = new();
-        controlDesigner.Initialize(button);
-        Assert.Empty(controlDesigner.AssociatedComponents);
-        Assert.False(controlDesigner.IsRootDesigner);
-        Assert.NotNull(controlDesigner.SnapLines);
-        Assert.Equal(8, controlDesigner.SnapLines.Count);
-        Assert.NotNull(controlDesigner.StandardBehavior);
-        Assert.Equal(Cursors.Default, controlDesigner.StandardBehavior.Cursor);
-    }
 
-    [Fact]
-    public void AccessibleObjectField()
-    {
-        using TestControlDesigner controlDesigner = new();
-        using Button button = new();
         controlDesigner.Initialize(button);
-        Assert.Null(controlDesigner.GetAccessibleObjectField());
+
+        controlDesigner.AssociatedComponents.Count.Should().Be(0);
+        controlDesigner.IsRootDesigner.Should().BeFalse();
+        controlDesigner.SnapLines.Should().NotBeNull();
+        controlDesigner.SnapLines.Count.Should().Be(8);
+        controlDesigner.StandardBehavior.Should().NotBeNull();
+        controlDesigner.StandardBehavior.Cursor.Should().Be(Cursors.Default);
     }
 
     [Fact]
-    public void BehaviorServiceProperty()
+    public void InitializeControlDefaults()
     {
-        using TestControlDesigner controlDesigner = new();
-        using Button button = new();
-        controlDesigner.Initialize(button);
-        Assert.Null(controlDesigner.GetBehaviorServiceProperty());
+        _designer.GetAccessibleObjectField().Should().BeNull();
+        _designer.GetBehaviorServiceProperty().Should().BeNull();
+        _designer.AccessibilityObject.Should().NotBeNull();
+        _designer.GetEnableDragRectProperty().Should().BeFalse();
+        _designer.ParticipatesWithSnapLines.Should().BeTrue();
+        (_designer.AutoResizeHandles = true).Should().BeTrue();
```

Comment:
I think this will be clearer:
```suggestion
        _designer.AutoResizeHandles = true;
```

If this assignment fails, the assert on the next line will fail too, am I missing anything?

Reply:
Do we want to test the property value before the assignment as well?

Reply:
Since this method is called `InitializeControlDefaults`, I'd prefer to just assert that `AutoResizeHandles` is false by default here and test the setter in a new method below it.